### PR TITLE
Add Capistrano

### PR DESCRIPTION
**Pull request by davinlagerroos**
Add capistrano for this app.

### CODE REVIEW

File: config/deploy/production.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+role :app, %w{asr-course-production.oit.umn.edu}
+
+# Configuration
+set :user, 'swadm'
+set :server, 'asr-course-production.oit.umn.edu'
+set :roles, %w{app}
+set :app_root, '/swadm/apps/peoplesoft_course_class_data'
```
Comment (DebbieGillespie) [inline config/deploy/production.rb]: Since this is the same for both environments, would it be better to put app_root in the main deploy.rb file?

Comment (davinlagerroos) [inline config/deploy/production.rb]: Good suggestion. Should/could I move the `set :user`, `set :roles`, and `set :deploy_to` as well?

Comment (DebbieGillespie) [inline config/deploy/production.rb]: I believe you can.

Debbie

File: config/environments/staging.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# This is the same context as the environment.rb file, it is only
+# loaded afterwards and only in the production environment
+
+# Change the production log level to debug
+#config.log_level = :debug
```
Comment (DebbieGillespie) [inline config/environments/staging.rb]: Why do we need this file since it only contains comments?  Mainly curious.

Comment (davinlagerroos) [inline config/environments/staging.rb]: daemon kit expects it to be there when we start up the daemon with `DAEMON_ENV=staging`. This file does contain some configuration info in another branch, but I added it here so I could keep daemon-kit happy when I was testing deployments on staging

**Comment by IanWhitney** [comment]
ok

**Comment by DebbieGillespie** [comment]
:+1:

**Pull request merged**